pyscript.core/src/stdlib/pyweb/pydom.py

```diff
@@ -34,6 +36,23 @@ def JsProxy(obj):
 alert = window.alert
 
 
+class JSProperty:
+    """JS property descriptor that directly maps to the property with the same
+    name in the underlying JS component."""
+
+    def __init__(self, name: str, allow_nones: bool = False):
+        self.name = name
+        self.allow_nones = allow_nones
+
+    def __get__(self, obj, objtype=None):
```

Comment:
out of curiosity, what's `objtype` about?

Reply:
We can actually remove it. Here's an [answer](https://stackoverflow.com/questions/30434154/what-is-the-usage-of-third-argument-objtype-in-python-descriptors-get) (even though it's actually not in the official docs even)

Reply:
In the docs it is called `owner` and it is always the type of the instance. It is useful if the descriptor is accessed through the class itself (e.g. Foo.x) when instance will be None.